css/md_screen.css

```diff
@@ -0,0 +1,19 @@
+@media screen and (min-width: 720px) {
+  .header_logo {
```

Comment:
It is good that you have added the media queries as required but the class names are still not expressive. What do I mean? It is expected that I can have multiple styles for an element so that at different screen sizes the appropriate style will take effect. For instance, you have `.header_logo` for all breakpoints you should have` .small_header_logo`, `.medium_header_logo and large_header_logo` this allows the user of your framework to choose and pick how they want to style their own projects. Let me show how it could be used in HTML as well
`<img class = "large_header_logo small_header_logo">` this means i only want to style for large and small screen size
 I hope i have explained this very well